presto-main/src/main/java/com/facebook/presto/sql/planner/PlanVariableAllocator.java

```diff
@@ -163,33 +80,11 @@ public TypeProvider getTypes()
         return TypeProvider.viewOf(variables);
     }
 
-    private int nextId()
-    {
-        return nextId++;
-    }
-
     public VariableReferenceExpression toVariableReference(Expression expression)
     {
-        checkArgument(expression instanceof SymbolReference, "Unexpected expression: %s", expression);
```

Comment:
why did you change the %s?

Reply:
Not using guava methods anymore, I created my own utility

Reply:
Did not want to add guava dependency on spi